app/views/checkout.html

```diff
@@ -0,0 +1,35 @@
+<h2>Checkout</h2>
+<div ng-show="basket.count() > 0">
+  <section id="checkout">
+    <div id="order_items">
+      <table>
+        <tr>
+          <th>Item Name</th>
+          <th>Item Price</th>
+          <th>Item Quantity</th>
+          <th>Subtotal</th>
+          <th></th>
+        </tr>
+        <tr ng-repeat="item in basket.get()">
+          <td>{{item.name}}</td>
+          <td>{{item.price | currency}}</td>
+          <td>{{item.quantity}}</td>
+          <td>{{item.quantity * item.price | currency }}</td>
+          <td><a href="" ng-click="basket.remove(item)">Remove</a></td>
```

Comment:
Yes this was intentional. I usually have to add `href=""` when using `ng-click` on an anchor tag in order for the pointer to change accordingly when hovering on the link.

Have a better alternative to doing this?